client/js/components/snippets/snippet.jsx

```diff
@@ -124,22 +130,25 @@ export default class Snippet extends React.Component {
       mime: this.props.language
     };
     let { style } = this.props;
-    let author = this.props.user.name;
-
+    let currentUser = this.getCurrentUser();
+    let author = this.props.user || currentUser;
+    let authorName = (author && author.name) || 'author';
```

Comment:
@vrael560 @blimusiek i'd leave this for now (doesn't create any problems i guess) and delete once session problem is resolved